graphql/schema.graphql

```diff
@@ -404,6 +457,27 @@ input DateInput {
 	year: Int!
 }
 
+type DateTime {
```

Comment:
Is there a reason you aren't using/modifying the existing date datatype?

https://github.com/NEU-DSG/dailp-encoding/blob/f5ca41369dbf251e714a48d9ea4fbbdf240d7cd4/graphql/schema.graphql#L382

Reply:
Date wraps a "Date," the new DateTime type wraps a "DateTime"